### ISSUE TITLE
new filter management: need a quick way to see who's in a filter without scrolling lists

### ISSUE THREAD

**Issue by rahaeli** [dreamwidth/dw-free#3252]
Usability issue for people with large access lists or many filters: there's no easy way to see just who's in the filter without having to skim through the entire pane of group members and look for the checkmarks (which are easy to miss). There needs to be some kind of way to quickly see only the people who are in the filter, whether that's sorting the people whose checkboxes are checked to the top of the "group members" list (which is not my ideal solution but would probably do) or adding another box in between the drop-down to choose the filter and the "Group Members" box that only contains all of the people in that actual filter.

**Comment by rahaeli** [comment]
Okay, I played around with the page for like an hour and I only just now noticed the "only show selected" button while I was reviewing screenshots. So, the thing exists, it's just ... really not obvious. Maybe it would be more obvious if labeled "only show members"? IDK, but my brain literally did not see the button until just now.

**Comment by momijizukamori** [comment]
Yeah I'm open to feedback on the controls for that, I wasn't sure how to fit everything in and still have it be clear.

**Comment by rahaeli** [comment]
Yeah, the problem is I don't have a hell of a lot of ideas either :/
